src/vim.h

```diff
@@ -598,7 +598,7 @@ extern int (*dyn_libintl_wputenv)(const wchar_t *envstring);
 # ifdef bindtextdomain
 #  undef bindtextdomain
 # endif
-# define bindtextdomain(x, y) // empty
+# define bindtextdomain(x, y) (y)
```

Comment:
```suggestion
# define bindtextdomain(x, y) ""
```
This is better as `bindtextdomain(<whatever>, NULL)` does not return `NULL` (unless there's something wrong).